ui/src/components/view/Metrics.vue

```diff
@@ -0,0 +1,196 @@
+      </a-col>
+      <a-col :md="24" :lg="12" :gutter="12">
+        <a-card>
+          <a-table
+            class="metric-card"
+            :columns="columns"
+            :data-source="usageMetrics"
+            :pagination="false"
+            size="middle"
+          >
+            <template slot="title">
+              <a>{{ $t('label.desc.usage.stats') }}</a>
+            </template>
+          </a-table>
+        </a-card>
+      </a-col>
+    </a-row>
+  </div>
+</template>
+
+<script>
+import { api } from '@/api'
+import { genericCompare } from '@/utils/sort.js'
+import Breadcrumb from '@/components/widgets/Breadcrumb'
+
+export default {
+  name: 'Metrics',
+  components: {
+    Breadcrumb
+  },
+  props: {
+    loading: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      dbMetrics: [],
+      usageMetrics: [],
+      columns: [
+        {
+          title: this.$t('label.name'),
+          sorter: true,
+          dataIndex: 'name',
+          width: '30%'
+        },
+        {
+          title: this.$t('label.value'),
+          dataIndex: 'value'
+        }
+      ]
+    }
+  },
+  mounted () {
+    this.fetchDetails()
+  },
+  watch: {
+  },
+  methods: {
+    fetchDbMetrics () {
+      var metrics
+      api('listDbMetrics').then(json => {
+        metrics = this.mapToArray(json.listdbmetricsresponse.dbMetrics)
+        this.dbMetrics = metrics
+      })
+      return metrics
+    },
+    fetchUsageMetrics () {
+      var metrics
+      api('listUsageServerMetrics').then(json => {
+        metrics = this.mapToArray(json.listusageservermetricsresponse.usageMetrics)
+        this.usageMetrics = metrics
+      })
+      return metrics
+    },
+    mapToArray (map) {
+      /* eslint-disable no-unused-vars */
+      var array = []
+      for (var key in map) {
+        if (key === 'replicas') {
+          map[key].forEach(function (value, i) {
+            var metric = {}
+            var name = 'replica[' + i + ']'
+            metric.name = name
+            metric.value = value
+            array.push(metric)
+          })
+        } else if (key === 'loadaverages') {
+          map[key].forEach(function (value, i) {
+            var metric = {}
+            metric.name = 'load-average[' + i + ']'
+            metric.value = value
+            array.push(metric)
+          })
+        } else {
+          var metric = {}
+          metric.name = key
+          metric.value = map[key]
+          array.push(metric)
+        }
+      }
+      /* eslint-enable no-unused-vars */
+      return array
+    },
+    fetchDetails () {
+      this.dbMetrics = this.fetchDbMetrics()
+      this.usageMetrics = this.fetchUsageMetrics()
+    },
+    fetchUsageListData () {
+      this.columns = []
+      this.columns.push({
+        dataIndex: 'name',
+        title: this.$t('label.name'),
+        sorter: function (a, b) { return genericCompare(a[this.dataIndex] || '', b[this.dataIndex] || '') }
```

Comment:
Sorting didn't work for me in my test

Reply:
it was an experiment and is not supposed to